src/platform/audio/ffmpegaudiofilecache.cpp

```diff
@@ -324,9 +334,11 @@ void *FFmpegAudioFileCache::Open_File(AudioEventRTS *audio_event)
         case 3:
             return nullptr;
         default:
-            break;
+            return nullptr;
```

Comment:
Why does it return here now?

Reply:
Well, the break here makes no sense, but i can revert it